content/tutorials/jwt-authentication.textile

```diff
@@ -136,96 +139,97 @@ This step shows you how to create the server code from scratch. If you get stuck
 npm install jsonwebtoken express
 ```
 
-3. Copy your API key to the clipboard, from your Ably Dashboard. It will be of the form @12AB3a.ab3GOm:c-B4Emryq9aFmOxP@.
+3. Copy your API key to the clipboard, from your Ably Dashboard. It will be of the form:
```

Comment:
I think it might be worth breaking down the individual parts of the key here, especially as you do `split()` on it in step 2.7?

```suggestion
3. Copy your API key to the clipboard, from your Ably Dashboard. It will be of the form @<app ID>.<key ID>:<key secret>@. For example: @12AB3a.ab3GOm:c-B4Emryq9aFmOxP@.
```